openfecwebapp/templates/legal-doc-search-results.html

```diff
@@ -3,62 +3,62 @@
-              {% endif %}
-              {% if results.offset + results.limit < results.count %}
-              <a class="paginate_button next" href="/legal/advisory-opinions?search={{ query }}&offset={{ results.offset + results.limit }}">Next</a>
-              {% else %}
-              <span class="paginate_button next is-disabled">Next</span>
-              {% endif %}
+  <section class="main__content--full">
+    {% include 'partials/legal-filter.html' %}
+        <div id="results-{{ result_type }}" class="dataTables_wrapper dataTables_wrapper--panel">
+          <div class="results-info results-info--top">
+            <div class="results-info__left">
+              <h2 class="results-info__title">Searching {{ document_type_display_name }}{% if query %} for “{{ query }}”{% endif %}</h2>
             </div>
           </div>
-        {% else %}
-          <div class="message message--alert">
-            <p>Sorry, there were no results.</p>
-          </div>
-        {% endif %}
-      </div>
-    </section>
-    </div>
-  </div>
-</div>
+          {% if results.count %}
+            <table class="data-table data-table--text">
+              <thead>
+                <tr>
+                  <th class="cell--15">No.</th>
+                  <th class="cell--25">Name</th>
+                  <th class="cell--60">Hit</th>
+                </tr>
+              </thead>
+              <tbody>
+                {# TODO this should handle generic document types (aos, murs, regs, etc) #}
+                {% for advisory_opinion in results.results %}
+                  {% include 'partials/legal-search-results-advisory-opinion.html' %}
+                {% endfor %}
+              </tbody>
+            </table>
+            <div class="results-info results-info--bottom">
+              <div class="dataTables_info">Showing {{ results.offset + 1 }}&ndash;{{ results.offset + results.results|length }} of about {{ results.count }}</span></div>
+              <div class="dataTables_paginate">
+                {% if results.offset > 0 %}
+                <a class="paginate_button previous" href="{{ url_for('advisory_opinions', search=query, offset=(results.offset - results.limit)) }}#results-{{ result_type }}">Previous</a>
+                {% else %}
+                <span class="paginate_button previous is-disabled">Previous</span>
+                {% endif %}
+                {% if results.offset + results.limit < results.count %}
+                <a class="paginate_button next" href="{{ url_for('advisory_opinions', search=query, offset=(results.offset + results.limit)) }}#results-{{ result_type }}">Next</a>
+                {% else %}
+                <span class="paginate_button next is-disabled">Next</span>
+                {% endif %}
+              </div>
+            </div>
+          {% else %}
+            <div class="message message--alert">
+              <p>Sorry, there were no results.</p>
```

Comment:
Just flagging that this message should be updated to match the new alert pattern that we're implementing in https://github.com/18F/fec-style/pull/337